### DISCUSSION TITLE
Should css and js folders be committed to git?

### DISCUSSION THREAD

**Discussion by wimdeblauwe** [tailpress/tailpress#169] [General]
I created my first tailpress theme with the `--git` option and I see that there is `/resources/css` and `/resources/js`, but also `/css` and `/js`. I believe the ones in `/resources` is what I can edit and the others are just output. Shouldn't they be added to `.gitignore` so they are not committed to git?

**Comment by jeffreyvr** [comment]
I think that is a personal preference. It depends a little bit on how you distribute your theme.

**Reply by wimdeblauwe** [reply]
ok, I added this to my .gitignore:

```
/css
/js
*.zip
```

And I also added these scripts to generate a zip file with the theme to easily install it on a Wordpress site:

```
		"zip": "bestzip my-custom-theme.zip screenshot.png *.php style.css css/* js/* images/* template-parts/*",
		"production-zip": "npm run production && npm run zip",
```

Note that you need to run to install `bestzip` for this to work:

```
npm install --save-dev bestzip
```

**Reply by ReyJust** [reply]
Thanks! I'll use that!

There is also the `/fonts` dir if you are uploading custom fonts locally. Your util could be such as:
```
"zip": "bestzip my-custom-theme.zip screenshot.png *.php style.css css/* js/* fonts/* images/* template-parts/*",
```
